sydent/sydent.py

```diff
@@ -66,6 +66,10 @@
         'log.path': '',
         'log.level': 'INFO',
         'pidfile.path': 'sydent.pid',
+        # Uncomment the following to enable prometheus and/or sentry support
```

Comment:
well no, you don't uncomment them, you add things to your local config file.

you've got two options: 
* either make the default value be `''`, which means 'disable', like `internalapi.http.port`
* or put a wordy description, a bit like `internalapi.http.bind_address`, only with more words.

Reply:
Sigh, today is not going well for me. 

Unfortunately `''` is a special value for prometheus package, e.g. setting `''` as bind addr makes it listen on everything